demo-java/app/build.gradle

```diff
@@ -81,3 +72,11 @@ dependencies {
     v3Implementation 'com.google.auto.value:auto-value-annotations:1.8'
     v3Implementation 'com.google.code.gson:gson:2.8.6'
 }
+
+secrets {
+    // To add your Places API key to this project:
+    // 1. Open the file local.properties
+    // 2. Add this line, where YOUR_API_KEY is your API key:
+    //        PLACES_API_KEY=YOUR_API_KEY
+    defaultPropertiesFileName 'local.defaults.properties'
```

Comment:
Do I need a Step 3 to change this value to 'local.properties'? When I leave it as 'local.defaults.properties' it looks for the values in the defaults file instead of my local.properties file.

Reply:
Actually I read the secrets-gradle-plugin README and it looks like I need to add a line
`propertiesFileName 'local.properties'`
so maybe that is step 3?

Reply:
> Actually I read the secrets-gradle-plugin README and it looks like I need to add a line propertiesFileName 'local.properties' so maybe that is step 3?

secrets-gradle-plugin will by default read values in `local.properties` so it's not necessary to have an entry for `propertiesFileName`. I can certainly add it still with a comment that it's not necessary but it's include in here for the sake of clarity. Thoughts on this?

Reply:
I see! yes, I think it will be helpful to state in Step 1 that that `local.properties` will be read by default by the plugin wtihout needing to declare it explicitly here. I was also unsure where in the file structure (I put it in the same parent folder as where local.defaults.properties exists) to create the file, so maybe instruction about that would help too (or does it just get found if stored anywhere in the project directories).